deb/build/build.sh

```diff
@@ -13,9 +13,15 @@ mkdir -p $D
 # so that multiple builds can go on concurrently
 cp -R $dir/* $D
 
+# Create a description temp file
+sed -i.bak -e 's/^\s*$/./' -e 's/^/ /' $DESCRIPTION_FILE
+
 # Expand variables in the definition
 "$BASE/bin/branding.py" $D/debian
 
+# Rewrite the file 
+mv "$DESCRIPTION_FILE.bak" "$DESCRIPTION_FILE"
```

Comment:
What's the point of creating the description temp file and then overwriting it with the original (original created with `sed -i.bak...`)?